rocketmq-knative/source/config/500-controller.yaml

```diff
@@ -0,0 +1,49 @@
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#   (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: rocketmqsource-controller-manager
+  namespace: knative-sources
+  labels:
+    control-plane: rocketmqsource-controller-manager
+spec:
+  selector:
+    matchLabels: &labels
+      control-plane: rocketmqsource-controller-manager
+  serviceName: rocketmqsource-controller-manager
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      serviceAccountName: rocketmqsource-controller-manager
+      containers:
+      - name: manager
+        image: docker.io/wlliqipeng/alitablestore-controller-test:0.5.0
```

Comment:
alitablestore-controller?

Reply:
sorry, the image is misquoted,I'll change it

Reply:
done